astroquery/first/tests/test_first.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
```

Comment:
we live in the python3 future, no need for this :)

Reply:
Made these changes too